scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/authentication/ManageUsers.java

```diff
@@ -428,17 +581,18 @@ private static void storeLoginFile(String loginFilePath, Properties props) throw
         for (Map.Entry entry : sortedUsers.entrySet()) {
             modifiedLines.add(entry.getKey() + ":" + entry.getValue());
         }
-        try (FileOutputStream writer = new FileOutputStream(loginFilePath)) {
-            IOUtils.writeLines(modifiedLines, System.getProperty("line.separator"), writer, Charsets.UTF_8);
+        try (BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(new FileOutputStream(loginFilePath)))) {
+            IOUtils.writeLines(modifiedLines, System.getProperty("line.separator"), writer);
         }
+        System.out.println("Stored login file in " + loginFilePath);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS106)